### PR TITLE
Update cable positions every move

### PR DESCRIPTION
**Pull request by KaiMikkelsen**

### CODE REVIEW

**Review by jatinchowdhury18 — COMMENTED** [review]
This is looking great! I just ran into one issue that I mentioned in the comments.

**Comment by KaiMikkelsen** [comment]
> This is looking great! I just ran into one issue that I mentioned in the comments.

Weird I can't seem to find that comment?

File: src/gui/pedalboard/cables/CableView.cpp [hunk 1]
```diff
@@ -141,12 +141,6 @@ void CableView::timerCallback()
         portGlow = false;
         repaint (getPortGlowBounds (portToPaint, scaleFactor).toNearestInt());
     }
-
-    for (auto* cable : cables)
-    {
-        cable->updateStartPoint();
-        cable->updateEndPoint();
-    }
```
Comment (jatinchowdhury18) [inline src/gui/pedalboard/cables/CableView.cpp:149]: So I'm running into an issue where the cable position doesn't update when it's being dragged, which I think is related to this code being removed. That said, we probably don't need to call this code every time the timer runs, so maybe we could do something like this:
```cpp
if (isDraggingCable)
    updateCablePositions();
```

Comment (KaiMikkelsen) [reply src/gui/pedalboard/cables/CableView.cpp:149]: Ah right, good idea thanks. I added this line in the latest change.

**Comment by jatinchowdhury18** [comment]
Yeah, this looks great!

**Pull request merged**